pkg/core/mpt/branch.go

```diff
@@ -0,0 +1,78 @@
+package mpt
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/crypto/hash"
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+const (
+	// ChildrenCount represents a number of children of a branch node.
+	ChildrenCount = 17
+	// LastChild is the index of the last child.
+	LastChild = ChildrenCount - 1
```

Comment:
I'm not sure about exporting these, they seem to be an implementation detail that probably no one cares about.

Reply:
Fixed.